tests/dashbio_demos/utils/app_wrapper.py

```diff
@@ -5,8 +5,9 @@
 
 def app_page_layout(page_layout,
                     app_title="Dash Bio App",
-                    app_github_url="",
```

Comment:
@mkcor  if this argument is renamed here, it should be refactored in `index.py` as well, this is what make the Integration tests fail on CircleCI

Reply:
Fixed in https://github.com/plotly/dash-bio/pull/149/commits/531a8885586fa794eb97be6394aa9f8a626d09e4.